addons/rose/addon/components/rose/header/brand/index.hbs

```diff
@@ -15,7 +15,10 @@
   {{/if}}
 
   {{#if @text}}
-    <h1 class='rose-header-brand-text'>{{@text}}</h1>
+    <Hds::Text::Display
+      @tag='h1'
+      class='rose-header-brand-text'
```

Comment:
None blocking comment: with the change to Hds::Text, is this class need?

Reply:
It will not be needed shortly. @lisbet-alvarez's [PR](https://github.com/hashicorp/boundary-ui/pull/2763) will remove this component but I didn't want to not change it just in case this is merged first or anything like that.

Reply:
Thanks! that makes sense to me 😉